lib/internal/Magento/Framework/Locale/Test/Unit/TranslatedListsTest.php

```diff
@@ -64,12 +73,15 @@ public function testGetOptionAllCurrencies()
     public function testGetOptionCurrencies()
     {
         $allowedCurrencies = ['USD', 'EUR', 'GBP', 'UAH'];
-
+		$selectedCurrencies = ['USD', 'EUR'];
+                
         $this->mockConfig->expects($this->once())
             ->method('getAllowedCurrencies')
             ->will($this->returnValue($allowedCurrencies));
 
-        $expectedResults = ['USD', 'EUR', 'GBP', 'UAH'];
+        $this->listsModel->selectedCurrencies = $selectedCurrencies;
```

Comment:
Please mock the return of selected currencies from the `scopeConfig`.

Reply:
ok. Along with the above comment